src/examples/Elmah.Io.ElasticSearch.Web/Controllers/ElmahController.cs

```diff
@@ -80,9 +80,8 @@ public override void ExecuteResult(ControllerContext context)
             var currentContext = GetCurrentContext(context);
 
             var httpHandler = factory.GetHandler(currentContext, null, null, null);
-            var httpAsyncHandler = httpHandler as IHttpAsyncHandler;
 
-            if (httpAsyncHandler != null)
```

Comment:
sure this should be removed?

Reply:
This is a enhancement due new C# version 7 feature called **pattern matching**
The old lines 83 (**cast**) and 85 (**if statement**) were merged with one expression in new 84 line using **pattern matching**!

`if(httpHandler is IHttpAsyncHandler httpAsyncHandler)` 

This statement do a cast and a null check in one time!